docs/guide/quickstart.qmd

```diff
@@ -0,0 +1,763 @@
+### Step 2: Setup paths and model configuration
+Reference the paths of the Ribasim installation and model directory and define the time period (2022-01-01 until 2023-01-01) for the model simulation:
+```python
+base_dir = Path("c:/Ribasim")
+model_dir = base_dir / "Crystal_Basin"
+data_path = model_dir / "data/input/ACTINFLW.csv"
+
+starttime = "2022-01-01"
+endtime = "2023-01-01"
+model = Model(
+    starttime=starttime,
+    endtime=endtime,
+    crs="EPSG:4326",
+)
+```
+### Step 3: Flow boundary nodes
+Crystal Basin consists of two inflow points, the tributary and the main Crystal river, we will call them ‘Minor’ and ‘Main’ respectively.
+In order to define the time series flow rate (m³/s) we read the discharge data from ACTINFLW.csv.
+This inflow data goes monthly from 2014 to 2023.
+However, for this exercise actual runtime is already defined in step 2.
+
+```python
+data = pd.read_csv(data_path, sep=";")
+data['sum']= data['minor']+data['main']
+#Average inflow and max. of the whole summed inflow data timeseries
+#From 2014 - 2023
+print('Average inflowQ m3/s:',data['sum'].mean())
+print('Average inflowQ m3/s:',data['sum'].max))
+
+model.flow_boundary.add(
+    Node(1, Point(0.0, 0.0), name='Main'),
+    [flow_boundary.Time(time=data.time, flow_rate=data.main,
+    )]
+)
+
+model.flow_boundary.add(
+    Node(2, Point(-3.0, 0.0), name='Minor'),
+    [flow_boundary.Time(time=data.time, flow_rate=data.minor,
+    )]
+)
+```
+### Step 4: Basin node (confluence)
+To schematize the confluence from the tributary we will use the Basin node.
+The node by itself portrays as a bucket with a certain volume of water and can be used for different purposes, such as a reservoir, a lake or in this case a confluence.
+@fig-confluence visualizes a cross section of the confluence point in our model.
+
+![Basin node concept for the confluence](https://github.com/user-attachments/assets/3a6b05d9-c535-4054-be92-c593a0c03d93){fig-align="left" #fig-confluence}
+
+@tbl-input1 shows the input data for the basin node profile.
+
++-----------+----------+
+| Area [m2] |Level [m] |
++===========+==========+
+| 672000    |     0    |
++-----------+----------+
+| 5600000   |     6    |
++-----------+----------+
+
+: Profile data for the basin node {#tbl-input1}
+
+The following code will be:
+```python
+model.basin.add(
+    Node(3, Point(-1.5, -1), name='Conf'),
+    [basin.Profile(area=[672000, 5600000], level=[0, 6]),
+     basin.State(level=[4]),
+     basin.Time(time=[starttime, endtime]),
+    ],
+)
+```
+### Step 5: Tabulated rating curve
+In the previous step we implemented a Basin node that functions as a confluence.
+Conceptually, the basin acts like a bucket of water, accumulating inflows and then releasing them.
+However, the model does not run if the basin is directly connected to the terminal node.
+This is because, for the model to function properly, we need to define a relation between the water level (h) in the basin and the outflow (Q) from the basin.
```

Comment:
```suggestion
This is because, for the model to function properly, we need to define a relation between the water level ($h$) in the basin and the outflow ($Q$) from the basin.
```